.github/workflows/ct-lint.yaml

```diff
@@ -0,0 +1,95 @@
+name: Lint and Test Charts
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    env:
+      remote_url: https://github.com/rancher/helm3-charts.git
+      # Chart-testing parameters
+      ct_remote: rancher-helm3-charts
+      ct_target_branch: master
+      ct_debug: false
+      ct_print_config: true
+      ct_validate_maintainers: false
+      ct_check_version_increment: false
+      ct_namespace: default
+      ct_chart_dirs: $(find charts -type d -mindepth 1 -maxdepth 1 | paste -sd ',')
+      ct_chart_repos: "\
+        stable=https://charts.helm.sh/stable,\
+        incubator=https://charts.helm.sh/incubator"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # chart-testing requires a full clone
+          fetch-depth: 0
+
+      - name: Set up Rancher remote
+        run: >
+          git remote add ${{ env.ct_remote }} ${{ env.remote_url }}
+          --fetch
+          --track ${{ env.ct_target_branch }}
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.0.1
+
+      - name: Run chart-testing [ct list-changed]
+        id: list-changed
+        run: >
+          changed=$(
+          ct list-changed
+          --remote ${{ env.ct_remote }}
+          --target-branch ${{ env.ct_target_branch }}
+          --chart-dirs ${{ env.ct_chart_dirs }}
+          )
+
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          else
+            echo "No chart changes detected, skipping tests."
+          fi
+
+      - name: Run chart-testing [ct lint]
+        if: steps.list-changed.outputs.changed == 'true'
+        run: >
+          ct lint
+          --remote ${{ env.ct_remote }}
+          --target-branch ${{ env.ct_target_branch }}
+          --debug=${{ env.ct_debug }}
+          --print-config=${{ env.ct_print_config }}
+          --validate-maintainers=${{ env.ct_validate_maintainers }}
```

Comment:
We require maintainers ?

Reply:
Nope, partner charts are the partner's responsibility so it is really up to them to set maintainers. So I disabled that check because we shouldn't fail the test if they decide to not have any maintainers listed in the chart's metadata.